`validate_idranges` target is broken

Trying to validate the ID range file yields an error from Ammonite:

```sh
$ sh run.sh make validate_idranges
amm ..scripts/validate_id_ranges.sc uberon-idranges.owl
Failed to resolve ivy dependencies:/tools/.coursier-cache
make: *** [Makefile:986: validate_idranges] Error 1
```

Ammonite is seemingly trying to download and store the dependencies of the `validate_idranges` script into `/tools/.coursier-cache`, which it cannot do without root privileges.

--- a/template/src/ontology/Makefile.jinja2
+++ b/template/src/ontology/Makefile.jinja2
@@ -1333,8 +1333,8 @@ normalize_src: $(SRC)
 	$(ROBOT) convert -i $< -f {% if 'obo' == project.edit_format %}obo --check false{% else %}ofn{% endif %} -o $(TMPDIR)/normalise && mv $(TMPDIR)/normalise $<
 
 .PHONY: validate_idranges
-validate_idranges:
-	amm $(SCRIPTSDIR)/validate_id_ranges.sc {{ project.id }}-idranges.owl
+validate_idranges: | $(TMPDIR)
+	COURSIER_CACHE=$(TMPDIR)/coursier-cache amm $(SCRIPTSDIR)/validate_id_ranges.sc {{ project.id }}-idranges.owl
 
 .PHONY: update_repo
 update_repo:

